src/foundry/foundry.js/config.d.ts

```diff
@@ -674,9 +674,9 @@ declare global {
       collection: ConstructorOf<Users>;
 
       /**
-       * @defaultValue `PlayerConfig`
+       * @defaultValue `UserConfig`
        */
-      sheetClass: ConstructorOf<PlayerConfig>;
+      sheetClass: ConstructorOf<UserConfig>;
```

Comment:
`TS2304: Cannot find name 'UserConfig'.`

Has anyone an idea where I missed an export or something else?

Reply:
`UserConfig` is a `DocumentSheet` extending class in `foundry.js` and is not typed yet. Feel free to just type this as `unknown` for now and add a `TODO` comment.

Reply:
Are you sure? `UserConfig` is a `DocumentSheet` defined in `foundry.js`. I guess the problem is the same as with `Roll` in the other PR.